core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -636,5 +638,23 @@ public static String convertMsToDate(long millis) {
     return dateFormat.format(new Date(millis));
   }
 
+  /**
+   * Converts milliseconds to clock time.
+   *
+   * @param millis milliseconds
+   * @return input encoded as clock time
+   */
+  public static String convertMsToClockTime(long millis) {
```

Comment:
isn't there a java util for this already using https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html?

Reply:
For startTimeMs (start time in milliseconds), I want to change it to Date, and CommonUtils.convertMsToDate already support it. 
  public static String convertMsToDate(long millis) {
    DateFormat dateFormat = new SimpleDateFormat(DATE_FORMAT_PATTERN);
    return dateFormat.format(new Date(millis));
  }

For upTimeMs(Alluxio last time in milliseconds), both Web and CLI use the convertMsToClockTime.
SimpleDateFormat is good to convert to Date, but not suitable for converting to  String.format("%d day(s), %d hour(s), %d minute(s), and %d second(s)", days, hours, mins, secs);